Python/bytecodes.c

```diff
@@ -255,10 +255,28 @@ dummy_func(
             value2 = PyStackRef_DUP(GETLOCAL(oparg2));
         }
 
+        family(LOAD_CONST, 0) = {
+            LOAD_CONST_IMMORTAL,
+        };
+
         pure inst(LOAD_CONST, (-- value)) {
             value = PyStackRef_FromPyObjectNew(GETITEM(FRAME_CO_CONSTS, oparg));
         }
 
+        inst(LOAD_CONST_IMMORTAL, (-- value)) {
+            PyObject *obj = GETITEM(FRAME_CO_CONSTS, oparg);
+            assert(_Py_IsImmortal(obj));
+            value = PyStackRef_FromPyObjectImmortal(obj);
+        }
+
+        replicate(4) inst(LOAD_INT, (-- value)) {
+            /* Tell code generator that this is a const load */
+            (void)FRAME_CO_CONSTS;
+            assert(oparg < 256);
```

Comment:
Would it make sense to check for `(-_PY_NSMALLNEGINTS <= oparg && oparg < _PY_NSMALLPOSINTS)` instead?

Reply:
`oparg` is unsigned, so it will always be `>= 0`.
I'll add an assert that `oparg < _PY_NSMALLPOSINTS`